src/app/showcase/doc/calendar/propsdoc.ts

```diff
@@ -223,7 +223,7 @@ import { Component, Input } from '@angular/core';
                     </tr>
                     <tr>
                         <td>dataType</td>
-                        <td>string</td>
+                        <td>'date' | 'string'</td>
```

Comment:
This not should be edit here, should be in `calendar.ts` and then use the command: `npm run apidoc` and this file will be generated automatically